test-network/prometheus-grafana/README.md

```diff
@@ -0,0 +1,47 @@
+# Description
+
+This sample provides an environment to display and capture metrics from the test-network in real time. It consists of a docker-compose file that starts a prometheus and grafana server setup configured to collect and display metrics for the test network.
+
+# Requirements
+
+This sample has been tested and is meant to be used on **linux** in order to fully benefit from its capabilities.
+You will need to have installed **docker-compose with version 1.29 or above** (note that this is higher than the v1.14 requirement requested for the test-network).
+
+# How to use
+
+1. Bring up the test-network -> **./network.sh up**
+2. Bring up the Prometheus/Grafana network in the test-network/prometheus-grafana -> **docker-compose up -d**
+3. Log in: type “localhost:3000” on your web browser -> username=“admin”, password=“admin” -> set a new password
+4. Browse dashboard and analyse results 
+   - The default dashboard "HLF Performances Review" can be found and displayed by hovering over the dashboard menu and clicking on the browse button.
+   ![picture alt]("https://user-images.githubusercontent.com/86831094/149115445-5e5f6d95-ecc3-4b46-aadb-5c01148770b3.png "Title is optional")
+   Once opened the dashboard adjust the timeframe on the top right to display the latest collected metrics and data
```

Comment:
I had to set the time frame in order to see metrics in my case. Last 30 minutes, last 15 mintues worked.